<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCEdit tech demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMCEdit">
    <property name="title">
     <string>MCEdit</string>
    </property>
    <addaction name="actionNew_World"/>
    <addaction name="actionOpen_World"/>
    <addaction name="actionShow_World_List"/>
    <addaction name="separator"/>
    <addaction name="actionClose_World"/>
    <addaction name="actionSave_World"/>
    <addaction name="actionSave_World_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit_MCEdit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_MCEdit"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <addaction name="menuMCEdit"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolsToolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="panelsToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew_World">
   <property name="text">
    <string>New World...</string>
   </property>
  </action>
  <action name="actionOpen_World">
   <property name="text">
    <string>Open World...</string>
   </property>
  </action>
  <action name="actionShow_World_List">
   <property name="text">
    <string>Show World List</string>
   </property>
  </action>
  <action name="actionClose_World">
   <property name="text">
    <string>Close World</string>
   </property>
  </action>
  <action name="actionSave_World">
   <property name="text">
    <string>Save World</string>
   </property>
  </action>
  <action name="actionSave_World_As">
   <property name="text">
    <string>Save World As...</string>
   </property>
  </action>
  <action name="actionExit_MCEdit">
   <property name="text">
    <string>Exit MCEdit</string>
   </property>
  </action>
  <action name="actionAbout_MCEdit">
   <property name="text">
    <string>About MCEdit 2...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
